Ubuntu 17.04 / CakePHP / Grunt w/SASS / Foundation

rmarrs opened the pull request:
- returned to Ubuntu 16.04
- fixed php-fpm sock names for PHP7
- implemented Grunt SASS & Uglify for CakePHP
- added full foundation & foundation icons for CakePHP
- added Wordpress install to https://192.168.50.4:8084/

Commit:
Upgraded Ubuntu to 17.04
Added CakePHP
Fixed php-fpm.conf files across all platforms to name the socket php7-fpm.sock instead of php5-fpm.sock

Commit:
fixed the .gitignore to only reference /cakephp

Commit:
We don't need to be Very Very Verbose.

Commit:
adding in composer updates / grunt watching / livereload

Commit:
Adding SASS / grunt watch tasks, etc

Commit:
Upgraded versions, also added custom.scss file

Commit:
fixing Foundation Javascript
Calls foundation on load
grunt watch start

Commit:
create SSL Certificates
Update all roles to https
added default Foundation script to webroot/js/custom.js
remove the create_ssl.sh script after it runs

Commit:
adding in the custom templates that got missed in develop

Commit:
added auto-redirect for https

Commit:
updating documentation.

Still resolving rc-local issues

Commit:
fixed auto-start of grunt by moving to a template file

Commit:
updated Readme.md to reflect https

Commit:
fixed default scss

Commit:
fixed sass modules failing

dirkaholic commented:
I really like this but I just don't understand why the upgrade to Ubuntu 17.04 is included here. I would keep such a change separated and in general I'd would like to support LTS versions only and 17.04 is none.

rmarrs commented:
You are correct, that shouldn't have been rolled up. I'll get that fixed and request another pull.

Commit:
updated the cakephp task to not fail if the cakephp directory already
exists. Allows a vagrant destroy / vagrant up refresh

rolled back Ubuntu to 16.04

rmarrs committed:
added wordpress installation task to 192.168.50.4:8084
upgraded packages, etc

rmarrs commented:
I've downgraded back to 16.04, and added a Wordpress installation to port 8084.

rmarrs committed:
removing unnecessary comments

rmarrs committed:
defaulted to all environments on

rmarrs committed:
update Readme.md

Commit:
removed system update - hangs

S1SYPHOS commented:
As this issue is quite old, I'd like to know if this PR will ever land. Otherwise, may I suggest moving forward to v20.04, as suggested in issue #42?

I would really like to see this dev-box advance :fox_face:
